packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1749,9 +1747,7 @@ class _TimePickerInputState extends State<_TimePickerInput> with RestorationMixi
 
   String? _validateMinute(String? value) {
     final int? newMinute = _parseMinute(value);
-    setState(() {
-      minuteHasError.value = newMinute == null;
-    });
+    minuteHasError.value = newMinute == null;
```

Comment:
I think the ideal scenario would be to revert the change here if possible—just to show that fixing this bug doesn't require migrating existing code.

Or if migrating _is_ required, we would need to [handle the breaking change](https://github.com/flutter/flutter/blob/main/docs/contributing/Tree-hygiene.md#handling-breaking-changes).

Reply:
+1, the setStates could be removed in a followup PR.